Tests/README.md

```diff
@@ -0,0 +1,27 @@
+Tests Reeadme 
+==========
+
+Test projects in here use the NUnit test runner using VS2017 or XS.
+
+Building UWP target with ARM will run on a Lumia 950XL or similar device.
+
+It does **not** include Windows Phone 8.1 as that targets 
```

Comment:
This sentence doesn't sound correct. Additionally, beyond VS2017 not supporting WP8.1, we don't (and it's highly unlikely we ever will) support it.